packages/docusaurus-theme-classic/src/theme/MDXPage/index.tsx

```diff
@@ -66,6 +74,15 @@ export default function MDXPage(props: Props): JSX.Element {
               </div>
             )}
           </div>
+          <EditMetaRow
+            className={clsx(
+              'margin-top--sm',
+              ThemeClassNames.blog.blogFooterEditMetaRow,
+            )}
+            editUrl=""
```

Comment:
😄 for consistency with docs/blog, we'd also want to support the editUrl.

I think we should add this in this PR too because both features are quite related in terms of the way to display the data on the UI.